js/main.js

```diff
@@ -1,35 +1,61 @@
 'use strict';
 
 {
-   
-  document.getElementById('btn').addEventListener('click',() =>{
-    const tbody=document.querySelector('tbody');
-    const todoComment = document.getElementById('comment');
-    const stateButton= document.createElement('button');
-    stateButton.textContent='作業中';
-    const delButton = document.createElement('button');
-    delButton.textContent ='削除';
-   
-    const todoList =[tbody.rows.length,todoComment.value,stateButton,delButton];
+  const todos = [];
+  const tbody = document.getElementById('tbody');
 
-    const tr =document.createElement('tr');
 
-    for(let i = 0 ; i < todoList.length ; i ++){
+  function displayTodo(todos) {
     
-        const td = document.createElement('td');
-
-        if(i >= todoList.length- 2){
-          td.appendChild(todoList[i]);
-          tr.appendChild(td);
-        }else{
-          td.textContent = todoList[i];
-          tr.appendChild(td);
-        }
-
+    while (tbody.firstChild){
+      tbody.removeChild(tbody.firstChild);
     }
 
+    todos.forEach((todo,i) => {
```

Comment:
`,`の後ろにはスペースを空けましょう。
他も同様です。
また、引数名で一文字はNGとされていますので、indexやidxなどにすると良いです。
```suggestion
    todos.forEach((todo, i) => {
```